# Biol 130

#### Central dogma of molecular biology
  * Information storage -> information carrier -> active cell machinery
  * DNA is transcribed into RNA to be passed on to proteins
  * Recall the important differences:
    * deoxyribose vs ribose sugar
    * thymine vs uracil
    * implications on stability of the nucleic acid
    * double vs single stranded
    * RNA sometimes resembles stranded protein structures
    * Many RNAs are not mRNA (messenger) for protein transcription
    * end of recall
  * Central dogma is a template driven synthesis
  * One strand of DNA is used as a template strand for RNA
  * Complementary strand is the non-template strand
  * The sequence in the NON-template strand is the coding strand
  * Complementary base-pairing is done in an antiparallel fashion

#### Enzyme catalysis
  * RNA polymerase
  * splits the DNA strands and opens active site
  * lowers activation at the active site
  * moves along the DNA for the section of the template strand

#### Prokaryotic transcription
  * RNApol
  * sigma factor + core enzyme -> holoenzyme
  * sigma recognizes the start and binds to the -10 and -35 position
  * core does the catalysis
  * sigma factor recognizes the promoter sequence at -10 and -35 box
  * sigma factor positions the core enzyme and releases the core enzyme
  * a single core enzyme, and many different sigma factors
  * End sequence forms an RNA with a hairpin loop which dissociates the polymerase
  * Promotor strand is on the non-template strand
  * A single mRNA and encode multiple proteins, e.g. a gene cluster
  * Polycistronic mRNA encodes all genes for multiple proteins corresponding to multiple genes

#### Eukaryotic transcription
  * different because of high levels of compaction of DNA
  * Eukaryotes have 3 RNApols I II and III
  * One promoter region called (TATA box) at the -30 position
  * TATA box Binding Protein (TBP) binds to a TATA box (5' TATAAA 3')
  * Transcription factor (TFIID) binds to the TBP and allows for the other factors to 'pile on'
  * TFIIA TFIIB TFIIC etc join the transcription initiation complex and recruits a RNApol
  * TBP distorts the DNA by causing a kink
  * mRNA then undergoes processing before shipping to translation
  * 7-methylguanosine and triphosphate bridge to the 5' end (7-methylguanosine capping signal for mRNA)
  * poly(A) tail at the 3' end to extend half-life by allowing A tail to slowly degrade before genes
  * extrons vs introns -> extrons are coding regions and introns are non-coding regions
  * introns are spliced out of the (matured) transcript
  * carried out by 5 snRNPs and 100+ proteins that form sliceosomes a type of ribozyme
  * branch point adenine attacks a 5' splice site and breaks the boundary between the intron and exon then allows the exons to condense
  * number of exons in polycistronic mRNA spans a very large range average 8
  * From a single gene, can produce different proteins in different cells

#### Disadvantages of DNA splicing
  * Extremely energy intensive
  * Synthesizing very large molecules and trimming down to very small molecules
  * Each step could cause a mutation

#### mRNA QA processing
  * as the mRNA is exiting proteins check the mRNA before it leaves the nucleus
  * cap-binding protein, poly-A-binding protein check for processing at the front and end
  * exon junction complex are used to denote proper exon inclusion and intron exclusion
  * Nuclear pore complex checks for the 'QA proteins' on the mRNA
  *  
